docs/_data/portland-2024-config.yaml

```
@@ -107,23 +107,23 @@ date: # how do we handle these? human readable would be nice.
 
   # Conference is special for the index to show the combined conference days
   conference:
-    event: Main Conference
+    event: Speaker talks
```

This does not roll of the tongue. How about?

```suggestion
    event: Scheduled talks
```

---

Yea, I didn't know a great way to frame it. I was thinking something like `Speaker presentations`? Scheduled is not super valuable to include.

---

Programmed talks? Or even just Schedule?